src/main/scala/nl.knaw.dans.easy.bagstore/server/StoresServletComponent.scala

```diff
@@ -160,6 +160,7 @@ trait StoresServletComponent extends DebugEnhancedLogging {
               "Location" -> externalBaseUri.resolve(s"stores/$bagstore/bags/${ fileSystem.toUri(bagId).getPath }").toASCIIString
             )))
             .getOrRecover {
+              case e: CompositeException if e.throwables.exists(_.isInstanceOf[BagIdAlreadyAssignedException]) => BadRequest(e.getMessage) //TODO this is little hacky, can't this be done in a nicer way?
```

Comment:
A minor improvement
```suggestion
               case e: CompositeException(errors) if errors.exists(_.isInstanceOf[BagIdAlreadyAssignedException]) => BadRequest(e.getMessage) //TODO this is little hacky, can't this be done in a nicer way?
```

Reply:
Better drop this case, keep 165 and somehow a findFirst rather than collectResults in 
https://github.com/DANS-KNAW/easy-bag-store/blob/f0f11de6a2f7d50fbf618de2a6bc46bfb32c61f4/src/main/scala/nl.knaw.dans.easy.bagstore/component/BagStoresComponent.scala#L106-L116